package.json

```diff
@@ -31,8 +31,10 @@
     "build:prod": "webpack --config webpack.prod.js --mode production",
     "prepare": "npm run build:dev  && npm run build:prod",
     "postinstall": "npm run build",
-    "jest": "npx jest --testPathIgnorePatterns=render.test.js",
+    "jest": "npx jest --testPathIgnorePatterns=render.test.js --maxWorkers=50%",
+    "jest:ci": "npx jest --testPathIgnorePatterns=render.test.js --runInBand",
```

Comment:
Why?

Reply:
I also benched the tests using https://github.com/sharkdp/hyperfine and these are the results that I got. Over the course of 3-4 runs, `--maxWorkers=50%` seemed to be faster

![Screenshot 2023-06-20 at 8 46 43 PM](https://github.com/3dmol/3Dmol.js/assets/23498248/52bdb0a0-5044-4ce1-99d3-19a50651bc4a)